templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
We actually didn't want to use the `etcd.client_cert` because we wanted separate etcd client certs for our components so that in the event of a leaked cert, we only need to roll the respective components. For example, if Doppler & Metron shared the same etcd client certs and if it were compromised, then we would have to roll the entire CF and Diego deployment.

Also there is another job called the `syslog_drain_binder`. We didn't add that in this template because it could get pulled in from the stub. So in the stub we would have an additional property block like below:

```
syslog_drain_binder:
  etcd:
    client_cert: foo
    client_key: bar
```